r/R/util.R

```diff
@@ -215,3 +215,138 @@ handle_csv_read_error <- function(e, schema, call) {
 is_compressed <- function(compression) {
   !identical(compression, "uncompressed")
 }
+
+parse_period_unit <- function(x) {
```

Comment:
Yeah this is a good idea! I'll add a comment. Seems especially important in this case because there are some slightly counterintuitive design choices I've had to make in order to mirror lubridate 😁